src/xbot/common/controls/sensors/nav6/Nav6Gyro.java

```diff
@@ -16,6 +17,7 @@
     private SerialPort imuPort;
     private IMU coreSensor;
     private ContiguousHeading yawValue;
+    AHRS ahrs;
```

Comment:
Andy, the nav6 gyro is represented by the `IMU coreSensor` above, in the same way the NavX is represented by `AHRS`.